application/view/common/advanced-search/item-sets.phtml

```diff
@@ -30,6 +33,7 @@ if (!$ids) {
                 ],
                 'options' => [
                     'disable_group_by_owner' => $this->status()->isSiteRequest(),
+                    'lang' => ($filterLocale == 1 ? $lang : null),
```

Comment:
Just for simplicity/cleanliness it's probably best to just cast `filterLocale` to bool when getting it at the top of each of these files where it's used and thereby avoid all the `== 0`/`== 1` comparisons.

Reply:
OK updated in 5c6f968